speaking-portal-project/src/phonemeFactory/rhubarb.ts

```diff
@@ -35,26 +35,23 @@ export async function rhubarbProcessor(
         '-d',
         `.${text_file_name}`,
         '--extendedShapes',
-        'GX',
+        'GHX',
         `.${audio_file_name}`,
     ]
 
     const rhubarbProc = spawnSync('./rhubarb', args)
 
-    if (rhubarbProc.stderr) {
-        // throw Error(`${rhubarbProc.stderr}`)
-        // Rhubarb failures are tagged with [Fatal], marking a failed attempt to run, log these.
-        if (rhubarbProc.stderr.includes(`[Fatal]`)) {
-            throw Error(`${rhubarbProc.stderr}`)
-        }
-    }
     // Return to original directory
     try {
         chdir('../')
     } catch (err) {
         console.log(`Error message: ${err}`)
     }
 
```

Comment:
I also reordered this - in the case of an error with rhubarb I think we would end up stuck in the rhubarb directory otherwise